Tooltip bug fix

R-Tomas-Gonzalez opened the pull request:
Fixes #2408 

### What changes did you make and why did you make them ?
  - Instead of making a global change of '!important' on the wins-card, I went ahead and added in the '.project-card' class without the overflow style. This makes the '.wins-card' overflow style take precedence, without creating a global '!important' rule.
  - This also makes it so that this change does not effect any other page stylings using the ".project-card" or ".wins-page" attributes

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>
<img width="637" alt="Screen Shot 2022-01-23 at 2 20 13 PM" src="https://user-images.githubusercontent.com/63771558/150696613-452106b8-4f55-4096-9921-7a434b52349d.png">
</details>

<details>
<summary>Visuals after changes are applied</summary>
<img width="637" alt="Screen Shot 2022-01-23 at 2 20 35 PM" src="https://user-images.githubusercontent.com/63771558/150696633-8b12e746-243d-436b-ab81-26c6dcab8493.png">
</details>

R-Tomas-Gonzalez committed:
created an style to the overflow visibility on wins-card attribute. Takes care of the hover effect with the tooltip.

R-Tomas-Gonzalez committed:
Reverted original change. Added 'project-card' styling.

Instead of making a global change of '' on the wins-card, I went ahead and added in the '.project-card' class without the overflow style. This makes the '.wins-card' overflow style take precedence, without creating a global '' rule.

JessicaLucindaCheng commented:
- Availability: 6 hours
- Review ETA: Thursday, January 27, 2022

SAUMILDHANKAR commented:
ETA: 1/27/22
Availability: 3 hrs, Thursday 3-6 PM

anthonysim commented:
ETA: 1/29/22
Availability: 3 hrs, Saturday 12-3 PM

anthonysim reviewed (dismissed):
@R-Tomas-Gonzalez, LGTM! I see the change allows the tool tip overflow from being hidden. Good job!

SAUMILDHANKAR commented:
Revised review ETA: 1/29/22
Availability: 3 hrs, Saturday 4-7 PM

SAUMILDHANKAR commented on _sass/components/_projects-page.scss:
```diff
@@ -32,7 +32,7 @@
     border-radius: 16px;
     box-shadow: 0 0 8px 0 rgba($color-black, 0.2);
     margin-bottom: 24px;
-    overflow: hidden;
```
@R-Tomas-Gonzalez Great job in making the changes. Tooltip looks good in my environment. Just curious if overflow should be removed from _projects-page.scss?

R-Tomas-Gonzalez replied:
Hey Saumil! You're totally right. I will put that back. Whoops!

R-Tomas-Gonzalez replied:
Went ahead and added the "!important" attribute to overflow on the .wins-card class in the the wins-page.scss file. I feel fine with this since we will not be using a new wins-card most likely ever except for on this page. And if we do, we wouldn't want that attribute to be hidden anyway.

R-Tomas-Gonzalez committed:
added attribute to wins-card

this should take care of the overflow once and for all. Just be aware that this will be for effective for ALL wins-card classes.

JessicaLucindaCheng commented:
- Availability: 3 hours
- Review ETA: Thu, Feb 3, 2022

JessicaLucindaCheng commented:
Sorry, going to delay reviewing this by one day. New availability and ETA below:
- Availability: 3 hours
- Review ETA: Fri, Feb 4, 2022

JessicaLucindaCheng reviewed (changes requested):
@R-Tomas-Gonzalez Sorry for the delay in reviewing your pr. Since there are concerns about using !important and not duplicating .project-card scss class, can `overflow: visible` be added to `.project-card.wins-card` but outside of the media query? It produces the same outcome. Here is a link to `.project-card.wins-card`: https://github.com/hackforla/website/blob/d8400dcaf4af7bfeb3af856f259d71038c617df1/_sass/components/_wins-page.scss#L37-L41.

R-Tomas-Gonzalez commented:
> @R-Tomas-Gonzalez Sorry for the delay in reviewing your pr. Since there are concerns about using !important and not duplicating .project-card scss class, can `overflow: visible` be added to `.project-card.wins-card` but outside of the media query? It produces the same outcome. Here is a link to `.project-card.wins-card`:
> 
> https://github.com/hackforla/website/blob/d8400dcaf4af7bfeb3af856f259d71038c617df1/_sass/components/_wins-page.scss#L37-L41
> 
> .

Great catch! Thank you.

R-Tomas-Gonzalez committed:
added overflow visible to project-card-wins-card style in winspage sccss.

JessicaLucindaCheng reviewed (approved):
@R-Tomas-Gonzalez Thanks for making the changes! It looks good to me now.
